src/website/src/pages/ec/components/table/docs/usage.mdx

```diff
@@ -3,4 +3,61 @@ title: Usage
 order: 1
 ---
 
-(Work in progress)
+import { Paragraph, Anatomy, Link } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  Tables are used to structure complex data in a way that makes it easier for
+  users to scan and digest content of interest.
+</Paragraph>
+
+## Anatomy
+
+<Anatomy
+  image="https://inno-ecl.s3.amazonaws.com/media/images/EC/Table/Table%20-%20Desktop.png"
+  srcset="https://inno-ecl.s3.amazonaws.com/media/images/EC/Table/Table%20-%20Mobile.png 598w, https://inno-ecl.s3.amazonaws.com/media/images/EC/Table/Table%20-%20Desktop.png 734w"
+  alt="Anatomy of table"
+  legend={{
+    items: [
+      {
+        color: '#404040',
+        label: 'mandatory',
+      },
+      {
+        color: '#004494',
+        label: 'optional',
+      },
+    ],
+  }}
+/>
+
+| Elements | Mandatory | Description              |
+| -------- | --------- | ------------------------ |
+| Headings | yes       | Columns headlines        |
+| Cells    | yes       | Actual data in the cells |
+
+## Do's
+
+- reduce the table width to fit the content
+- align content left to right within a cell
+- use a dash ( - ) in null data cells
+- adjusts the styling to render a small/large table
+
+## Don'ts
+
+- don't spell out numbers
+- don't use colour alone to convey information
+
+## When to use
+
+- use when you want to store and show large/small amounts of info, static data
+- use when you want to make comparisons
+
+## When not to use
+
+- when you don't have any relationship among content you want to present
+
+## Notes
+
+### Design
+
+When including text links, use a different color to highlight this - just bolding or underlining the text doesn’t provide enough of a visual cue
```

Comment:
Each sentence above is not capitalised, maybe this last one should follow the same lowercase logic?